src/tunacode/tools/utils/ripgrep.py

```
@@ -88,10 +104,12 @@ def get_ripgrep_binary_path() -> Path | None:
         bundled_path = vendor_dir / binary_name
 
         if bundled_path.exists():
+            ripgrep_cache.set_binary_path(bundled_path)
             return bundled_path
     except Exception:
         pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)